include/vapor/FlowParams.h

```diff
@@ -53,35 +53,94 @@ class PARAMS_API FlowParams : public RenderParams {
 
     static std::string GetClassType() { return ("FlowParams"); }
 
-    // True  == Steady; False == Unteady
+    //! Sets the type of flow rendering algorithm being used.
+    //! Steady flow (streamlines) renders time-invariant trajectories that follow a vector field at a single timestep.
+    //! Unsteady flow (pathlines) render time-variant trajectories that advect through the timeseries of a loaded dataset.
+    //! \param[in] bool - Steady/streamlines = 1, Unsteady/pathlines = 0
     void SetIsSteady(bool steady);
+    
+    //! Gets the type of flow rendering algorithm being used.
+    //! Steady flow (streamlines) renders time-invariant trajectories that follow a vector field at a single timestep.
+    //! Unsteady flow (pathlines) render time-variant trajectories that advect through the timeseries of a loaded dataset.
+    //! \retval bool - Steady/streamlines = 1, Unsteady/pathlines = 0
```

Comment:
For these longer setter/getter documentation strings would it make sense to make use of \copydoc to avoid replication and improve maintainability? E.g. Document the setter (or getter) with the long string, and use \copydoc for the getter (or setter).

Reply:
@clyne I don't think \copydoc is what we'd want here.  If SetIsSteady(bool) used used \copydoc on this getter, the first word "Gets" would be applied.  But it may be an option to add a \brief and \details section to these functions.  IE:

//! \brief Gets the type of flow rendering algorithm being used.
//! \details Steady flow (streamlines) renders time-invariant trajectories that follow a vector field at a single timestep.
//! Unsteady flow (pathlines) render time-variant trajectories that advect through the timeseries of a loaded dataset.
bool GetIsSteady() const;

//! \brief Sets the type of flow rendering algorithm being used.
//! \copydetails FlowParams::GetIsSteady()
void SetIsSteady(bool);

Reply:
That makes sense. Maybe \sa (see also) would be an even better choice than \copydoc?

Reply:
Hmm I'm not sure how \sa would work here.  Check out FlowParams::Get/SetSteadyNumOfSteps().  I think \details and \copydetails does what we're looking for.

Reply:
Agreed!